phylo/src/likelihood/likelihood_tests.rs

```diff
@@ -44,18 +43,19 @@ fn setup_simple_phylo_info(blen_i: f64, blen_j: f64) -> PhyloInfo {
 #[test]
 fn dna_simple_likelihood() {
     let info = setup_simple_phylo_info(1.0, 1.0);
-    let model = DNASubstModel::new(DNA(JC69), &[]).unwrap();
-    let likelihood = DNALikelihoodCost { info: &info };
+    let model = DNASubstModel::new(JC69, &[]).unwrap();
```

Comment:
optional: Since you might create your model with various options (including an empty smth), it feels like using builders might be a good option. So it would look like
```rust
let model = DNASubstModelBuilder::new(JC69)::build();
```

Reply:
This is a good idea, but I need to think about how to make it work properly as an API for when the code comes together. This will happen soon, but not in this PR.